### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by Restent**
要睡觉了，之后慢慢翻

### CODE REVIEW

**Comment by Cubik65536** [comment]
已经转换为草稿，等翻译完成之后再做review

**Review by Cubik65536 — APPROVED** [review]
LGTM! 需要提高但是可以之后再做改进

File: en/faq.md [hunk 1]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 [[toc]]
 
-## 站点使用相关
+## Station usage related
 
-### 收不到验证邮件？
+### Can't receive verify email？
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your email's trash can at first. If you can't find verify email in trash can, please try send verify email again.
```
Comment (tnqzh123) [inline en/faq.md:13]: trash can 改成 trash folder 比较好，毕竟不是真正的“桶”，也有的邮箱把这个文件夹叫 Junk Email 的

File: en/faq.md [hunk 2]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
 
 [[toc]]
 
-## 站点使用相关
+## Station usage related
```
Comment (tnqzh123) [inline en/faq.md:9]: Website Related

这种短标题中超过三个字符的单词的首字母要大写

File: en/faq.md [hunk 3]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-### 收不到验证邮件？
+### Can't receive verify email？
```
Comment (tnqzh123) [inline en/faq.md:11]: 用 verification email，verify 是动词，这个句子里已经有一个 receive 了

File: en/faq.md (hunk 1 above)
Comment (tnqzh123) [reply en/faq.md:13]: verification email

File: en/faq.md [hunk 4]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-请先检查你的邮箱的垃圾箱。如果垃圾箱中也没有验证邮件，尝试再发送一次验证邮件。
+Please check your email's trash can at first. If you can't find verify email in trash can, please try send verify email again.
 
-如果仍然收不到验证邮件，请检查你的账号绑定的邮箱是否正确，如果不正确则将其更改为正确的邮箱。
+If you still can't receive verify email, please check your email that bind your account is correct, if incorrect, please change it to your correct email.
```
Comment (tnqzh123) [inline en/faq.md:15]: If you still cannot receive verification email, please check your whether email address that bind your account is correct. If not, please change it to the correct one.

File: en/faq.md [hunk 5]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-如果你确定你绑定的邮箱正确，并且验证邮件不在你的邮箱的垃圾箱里，请加入我们的 [官方用户交流群](/user-group.html)，联系站点管理员处理。
+If you sure your email that bind your account is correct, and verify email can't find in your trash can, please join our [User Group](/user-group.html), contact staff to process.
```
Comment (tnqzh123) [inline en/faq.md:17]: If you sure your email address bound to your account is correct, and the verification email cannot be found in the Junk Email folder, please join our [User Group](/user-group.html), contact staffs to address it.

File: en/faq.md [hunk 6]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-### 我的角色被别人占用了，我可以要回来吗？
+### My profile are have by others, I can get it back?
```
Comment (tnqzh123) [inline en/faq.md:19]: My profile has been taken by others, may I get it back?

File: en/faq.md [hunk 7]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-不可以。重名的几率很大，随意更改角色所有者对双方都不公平，所以不管你是名声多大的主播还是某个达官贵人家的孩子，我们都不会提供更改角色所有者的服务。
+No. The chance of having the same name is very high, change players' owner is unfair to both users, so whether you are an anchor or a rich people, we don't provide the service that change players' owner.
```
Comment (tnqzh123) [inline en/faq.md:21]: No. The chance of having the same name is very high, change profiles' owner is unfair to both users, so we won't change profiles' owner for everyone.

File: en/faq.md [hunk 8]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-如果你发现你的角色名被别人占用了，你可以先联系当前该角色的所有者进行协商，没准这事儿就成了，同时你还收获了一个朋友呢。如果你好好地和当前该角色的所有者商量（而不是一上来就用一种强硬的口气要求释放角色——这种情况大多不会有好结果），在大多数情况下，他们都可以是很好说话的。至于怎么联系？缘分到了就自然联系上了咯。
+If you find that your player's name has been occupied by others, you can contact this player's owner. If you talk to this player's owner properly (not use a strong tone to talk to this player's owner) , in many times, they are friendly. But how to contact? By fate.
```
Comment (tnqzh123) [inline en/faq.md:23]: occupied 改 taken，站里用的就是 taken

BTW 这句翻的很不错

File: en/faq.md [hunk 9]
```diff
@@ -6,23 +6,23 @@ sidebarDepth: 2
...
 
-有一个例外情况是你拥有正版 Minecraft 并且你的正版 ID 就是被他人占用的角色名。这种情况下，你可以通过 [绑定正版角色](/newbee/player.html#绑定正版角色) 来强制获取这个角色的所有权。
+There is a special case: you have Genuine Minecraft account, and your Genuine Minecraft ID just the player's name that be occupied by others. In this condition, you can pass [Bind your Genuine Minecraft Account](/newbee/player.html) to get this player's owner.
```
Comment (tnqzh123) [inline en/faq.md:25]: However, there is a special case: you have a Mojang account with Genuine Minecraft, and your Genuine Minecraft player name is just the player name that be taken by others. In this situation, you can get your profile by [Bind Your Mojang Account](/newbee/player.html).

**Review by tnqzh123 — CHANGES_REQUESTED** [review]
omg…这个翻译质量真的一言难尽，很多地方都有很浓的机翻味，不过也有的地方翻的不错

翻译是门技术活，有些东西是只可意会不可言传的，不能逐字逐句地根据字面意思翻译。可以根据语境对文本进行适当的修改，意思到了就行。

**Comment by tnqzh123** [comment]
可以多尝试用一些完成时、被动语态、各种从句这种在中国的英语考试里看起来很高大上的语法，这样作为文档这种比较 serious 的场合，内容看起来会更棒一点。（其实我也说不出到底是哪里更棒，可能是比较装逼吧）

不用担心用错，大胆用就行，大不了我给你 review 的时候改过来。其实这些在中国的英语考试里看起来很高大上的语法在以英语为主要受众的母语的社区中这些是非常常见的东西。

**Comment by Cubik65536** [comment]
我过两天抽时间改一下

**Comment by Restent** [comment]
其实有些单词我是从 Google Translate 抓过来的🌚
我先看看需要更改的内容

**Pull request closed without merging**